helios-testing/src/main/java/com/spotify/helios/testing/HeliosSoloDeployment.java

```diff
@@ -245,6 +247,17 @@ private String checkDockerAndGetGateway() throws HeliosDeploymentException {
     return gateway;
   }
 
+  private void pullIfAbsent(final String image) throws DockerException, InterruptedException {
+    try {
+      dockerClient.inspectImage(image);
+      log.info("helios-solo image {} is present. Not pulling it.", image);
```

Comment:
derp, how did we miss this? The log messages and `pull(PROBE_IMAGE)` are wrong. PR here to fix: https://github.com/spotify/helios/pull/918